README.md

```diff
@@ -111,6 +111,23 @@ For example, assuming the following directory bucket name `my-test-bucket--usw2-
 usw2-az1, then the URI used will look like: `s3://my-test-bucket--usw2-az1--x-s3/<PREFIX>` (**please note that the 
 prefix for Amazon S3 Express One Zone should end with '/'**), paired with region us-west-2.
 
+## Reduce Checkpoint Loss Risks with S3 Versioning
+When working with model checkpoints, users can accidentally overwrite or delete critical files. This can happen due to incorrect configuration or multiple hosts accessing the same storage path concurrently.
```

Comment:
I'd also consider advising against the async checkpointing interface, as it has race conditions

Reply:
Nice idea.